### DISCUSSION TITLE
Question about recent website UI update

### DISCUSSION THREAD

**Discussion by Kepler-16b** [civitai/civitai#415] [Q&A]
How do I turn draft into published state? Previously we just click the button and it's done. Now it's really confusing...

**Comment by parkkalmex678** [comment]
The process of turning a draft into a published state can vary depending on the platform or software being used. However, in general, you need to locate the "Publish" button or option and click on it to make your draft live.

For instance, if you're using a content management system for a website, you may need to navigate to the "Edit" page for your draft, review and revise it as needed, then click the "Publish" button or select "Publish" from a dropdown menu.

If you're experiencing difficulty with this process due to technical issues, professional IT support in Gastonia can help troubleshoot and resolve the problem. In case of damage to your computer systems or other equipment, such as from a fire, professional [fire damage restoration Gastonia](https://restoration1gastonia.com/fire-damage-restoration-gastonia/) can help you repair or replace damaged equipment, minimizing downtime and getting you back to work faster.
